recipe/meta.yaml

```
@@ -31,7 +31,7 @@ requirements:
     - pkg-config
   host:
     - python
-    - cython 0.29
+    - cython 3.0.0
     - extension-helpers
     - numpy {{ numpy }}
```

```suggestion
    - numpy 1.23 # [py==310]
    - numpy {{ numpy }} # [py>=310]
```
Because the cbc configuration currently yields 1.21 for 3.10, but a min version of 1.23 is specified upstream: https://github.com/astropy/astropy/blob/v6.1.0/pyproject.toml#L42

---

I think this logic is flawed. Can we change to the following?
`- numpy 1.23 # [py==310]`
 `- numpy {{ numpy }} # [py>=311]`